<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>bnp-make-frontend</string>
  </property>
  <property name="windowIcon">
   <iconset resource="bnp-make-frontend.qrc">
    <normaloff>:/images/bnp-make-frontend.ico</normaloff>:/images/bnp-make-frontend.ico</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidgetMenu">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabList">
       <attribute name="title">
        <string>List</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutList">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutList">
            <item>
             <widget class="QLabel" name="labelList">
              <property name="text">
               <string>BNP file:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditList">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonListBrowse">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonList">
            <property name="text">
             <string>List</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEditList">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSearch">
       <attribute name="title">
        <string>Search</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutSearch">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutSearchPath">
            <item>
             <widget class="QLabel" name="labelSearchPath">
              <property name="text">
               <string>Search path:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditSearchPath">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonSearchBrowse">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutSearchString">
            <item>
             <widget class="QLabel" name="labelSearchString">
              <property name="text">
               <string>Search string:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditSearchString"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonSearch">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEditSearch">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPack">
       <attribute name="title">
        <string>Pack</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutPack">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutPack">
            <item>
             <widget class="QLabel" name="labelPack">
              <property name="text">
               <string>Source path:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditPack">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonPackBrowse">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonPack">
            <property name="text">
             <string>Pack</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEditPack">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabUnpack">
       <attribute name="title">
        <string>Unpack</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutUnpack">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutUnpack">
            <item>
             <widget class="QLabel" name="labelUnpack">
              <property name="text">
               <string>BNP file:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditUnpack">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonUnpackBrowse">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonUnpack">
            <property name="text">
             <string>Unpack</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEditUnpack">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSettings">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutSettings">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutBnpMake">
            <item>
             <widget class="QLabel" name="labelSettingsBnpMake">
              <property name="text">
               <string>bnp_make path:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditBnpMake">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonBnpMakeBrowse">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutDataPath">
            <item>
             <widget class="QLabel" name="labelDataPath">
              <property name="text">
               <string>Data path:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditDataPath">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonDataPathBrowse">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTextEdit" name="textEditSettings">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="bnp-make-frontend.qrc"/>
 </resources>
 <connections/>
</ui>
